domain/schema/model/sql/0022-resource.sql

```diff
@@ -135,8 +142,17 @@ CREATE TABLE unit_resource (
     PRIMARY KEY (resource_uuid, unit_uuid)
 );
 
-CREATE TABLE resource_oci_image_metadata_store (
-    resource_uuid TEXT NOT NULL,
+CREATE TABLE resource_store (
```

Comment:
Would be good to comment what tables the store_uuid is the forgine key on (`resource_container_image_metadata_store` or `object_store_metadata`)